auslib/web/public/api.yml

```diff
@@ -0,0 +1,474 @@
+swagger: '2.0'
```

Comment:
I think you'll need to add this file to MANIFEST.in to make sure it ends up in built packages.